test/system/destination_test.go

```diff
@@ -368,8 +368,8 @@ func parseYAML(contents string) []*unstructured.Unstructured {
 func mc(args ...string) string {
 	command := exec.Command("mc", args...)
 	session, err := gexec.Start(command, GinkgoWriter, GinkgoWriter)
-	Expect(err).ToNot(HaveOccurred())
-	Eventually(session).Should(gexec.Exit(0))
+	ExpectWithOffset(1, err).ToNot(HaveOccurred())
```

Comment:
There's a function called `GinkgoHelper()` that you can call at the start of a test helper, and it will mean that test failures are reported on the line that called the helper, rather than in the helper - essentially the same as `...WithOffset(` but simpler. It comes into its own when a test helper calls a test helper, as you no longer need to calculate the offset yourself.